### PR TITLE
[NEW] Cloud Firewall Product Docs

### PR DESCRIPTION
**Pull request by rsyracuse**

### CODE REVIEW

File: docs/products/networking/cloud-firewall/get-started/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+You can assign the Firewall to more than one Linode service at a time. Continue the previous step to assign the Firewall to another Linode service.
+    {{</ note >}}
+
+1. Click on the **Add** button to assign the Firewall to your Linode(s).
+
+{{< note >}}
+If you have a Cloud Firewall attached to a Linode and you attempt to [migrate the Linode to a data center](/docs/guides/how-to-initiate-a-cross-data-center-migration-for-your-linode/) that does not support Cloud Firewalls, the migration fails.
+{{</ note >}}
+
+### Add New Cloud Firewall Rules
```
Comment (leslitagordita) [inline docs/products/networking/cloud-firewall/get-started/index.md:50]: Get Started is the shortest distance between starting and using the service. This section and the others can be removed from this page.

File: docs/products/networking/cloud-firewall/_index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Cloud Firewall
+description: "Linode Cloud Firewall offers a simple interface to protect your web apps. It is scalable security in seconds, allowing you to create custom firewall rules, making security more accessible."
+tab_group_main:
+    is_root: true
+    title: Overview
+    weight: 10
+cascade:
+    date: 2020-06-02
+    product_description: "Secure your network with Linode Cloud Firewall. Filter traffic by allowing it on selected ports and protocols."
```
Comment (leslitagordita) [inline docs/products/networking/cloud-firewall/_index.md:10]: Copy edit suggestion for clarity: "Secure your network with Linode Cloud Firewall. Create Firewall rules that only allow network traffic on selected ports and protocols."

File: docs/products/networking/cloud-firewall/_index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Availability
+
+Mumbai, India; Sydney, Australia;
+
+## Features
+
+### Simple Interface
+
+Control inbound and outbound traffic using the [Linode API](/docs/api/networking) or [Cloud Manager](https://www.linode.com/products/cloud-manager/).
```
Comment (leslitagordita) [inline docs/products/networking/cloud-firewall/_index.md:25]: The Linode CLI can be mentioned here too.

**Review by leslitagordita — APPROVED** [review]

File: docs/products/networking/cloud-firewall/_index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/products/networking/cloud-firewall/_index.md]: Can we suggest this for the marketing page as well - this was taken directly from the marketing copy: https://www.linode.com/products/cloud-firewall/

**Pull request merged**